pkg/controllers/collaset/synccontrol/update.go

```diff
@@ -333,17 +334,25 @@ type PodUpdater interface {
 }
 
 type GenericPodUpdater struct {
-	collaSet   *appsv1alpha1.CollaSet
-	ctx        context.Context
-	podControl podcontrol.Interface
-	recorder   record.EventRecorder
+	*appsv1alpha1.CollaSet
+	context.Context
```

Comment:
Maybe we can remove `context` this time.
In most of time, context is passed when method is called. It should not be carried as a object field.

Reply:
reasonable, ctx is removed from object filed and put into func params